LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Broad Institute, Inc.
```

Comment:
Do we need to update the years on this? If we do, it could be something like a range of years (1995-current year)

Reply:
If we do, we need to do it across all repos and services. See, for example, [Orchestration's license](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/LICENSE.txt). If we do, then I'd like to treat that as a separate task so all of our licenses are in sync.